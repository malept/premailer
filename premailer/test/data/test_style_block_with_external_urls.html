<html>
<head>
<title>Title</title>
<style type="text/css">
body {
  color:#123;
  background: url(http://example.com/bg.png);
  font-family: Omerta;
}
</style>
</head>
<body>
<h1>Hi!</h1>
</body>
</html>
