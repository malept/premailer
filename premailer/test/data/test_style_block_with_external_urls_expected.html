<html>
<head>
<title>Title</title>
</head>
<body style="color:#123;background:url(http://example.com/bg.png);font-family:Omerta">
<h1>Hi!</h1>
</body>
</html>
